Re-factor project organization and build to use Alfresco Maven SDK

jpotts opened the pull request:
Migrate from Ant-based build to Maven SDK 2.2.0.

wabson commented:
Hi @jpotts thanks for this PR 😄. I would prefer to use the SDK3 rather than 2.2 but this could be done later and this PR gets us at least half-way there and I can experiment with the 2->3 conversion on a branch.

The only issue I see with the build here is that the webscript and client-side assets are exploded inside the AMP file so I changed the build to package them inside the JAR. Can you see any issues with that?

I also added some changes to the README.

jpotts commented:
No strong opinion on the web script and client-side asset re-org.

Agreed on SDK3, but I'd recommend waiting until that's GA. I've done an
upgrade on another project from 2.2 to 3 and it wasn't bad.

On Thu, Feb 9, 2017 at 8:27 AM, Will Abson <notifications@github.com> wrote:

> Hi @jpotts <https://github.com/jpotts> thanks for this PR 😄. I would
> prefer to use the SDK3 rather than 2.2 but this could be done later and
> this PR gets us at least half-way there and I can experiment with the 2->3
> conversion on a branch.
>
> The only issue I see with the build here is that the webscript and
> client-side assets are exploded inside the AMP file so I changed the build
> to package them inside the JAR. Can you see any issues with that?
>
> I also added some changes to the README.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/share-extras/site-blog-dashlet/pull/5#issuecomment-278656282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABz-25WaRWxc2U9KuC_uTraE6v85mYIEks5rayJLgaJpZM4Lx_5s>
> .
>
